enterprise/cli/prebuilds_test.go

```diff
@@ -0,0 +1,343 @@
+package cli_test
+
+import (
+	"bytes"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/coder/coder/v2/cli/clitest"
+	"github.com/coder/coder/v2/coderd/coderdtest"
+	"github.com/coder/coder/v2/codersdk"
+	"github.com/coder/coder/v2/enterprise/coderd/coderdenttest"
+	"github.com/coder/coder/v2/enterprise/coderd/license"
+)
+
+func TestPrebuildsPause(t *testing.T) {
+	t.Parallel()
+
+	t.Run("Success", func(t *testing.T) {
+		t.Parallel()
+
+		client, _ := coderdenttest.New(t, &coderdenttest.Options{
+			LicenseOptions: &coderdenttest.LicenseOptions{
+				Features: license.Features{
+					codersdk.FeatureWorkspacePrebuilds: 1,
+				},
+			},
+		})
+
+		inv, conf := newCLI(t, "prebuilds", "pause")
+		var buf bytes.Buffer
+		inv.Stderr = &buf
+		//nolint:gocritic // Only owners can change deployment settings
+		clitest.SetupConfig(t, client, conf)
+
+		err := inv.Run()
+		require.NoError(t, err)
+
+		// Verify the output message
+		assert.Contains(t, buf.String(), "Prebuilds are now paused.")
+
+		// Verify the settings were actually updated
+		//nolint:gocritic // Only owners can change deployment settings
```

Comment:
Why the `//nolint:gocritic`? 🤔

Reply:
There's a rule to check that we test with non-owner roles as they're usually more appropriate. In this case, owner is the appropriate role.